CNAME

```diff
@@ -0,0 +1 @@
+istillhaveboobs.com
```

Comment:
Personal domain should not be merged here.
And i assume that people who owns a domain and let github serve his blog should have known how to add a CNAME file. 😄